### DISCUSSION TITLE
Change of Address option no longer visible.

### DISCUSSION THREAD

**Discussion by emcgal** [woocommerce/woocommerce#38883] [General]
Hi, 
I have tried to find something on this which appeared a couple of weeks ago on a client website after she was stung (by a chancer imo) because we have and want the default shipping in the basket but the option to update the address is gone.

This website is always 100% up to date as still in maintenance. I took a staging there and turned off everything. I even made a quick basic woo site just to see was it something that maybe on the client website blocking it but the option is still missing.

![image](https://github.com/woocommerce/woocommerce/assets/5911943/230e541c-9dab-4b9a-8955-e219cf74c45c)
But the Change Address is missing from the basket.
![image](https://github.com/woocommerce/woocommerce/assets/5911943/916c34bd-6963-47f2-8e9e-fb763a2340c0)

I checked on a website I know is out of date (owner looks after it now) and the option on theirs but none of my maintained websites. Most don't have a large volume of international shipping but this website does. 

Anyone got any suggestions as I am stumped. 

Thanks,
Emer

PS: That is just the basic woocommerce setup btw. Not the client site as I was trying to see if I could replicate it. It was working in the past when I originally built the website.

**Comment by coreymckrill** [comment]
Converted this to an issue: https://github.com/woocommerce/woocommerce-blocks/issues/10685

**Comment by mikejolley** [comment]
It's ok @coreymckrill, we already picked these issues up and created https://github.com/woocommerce/woocommerce-blocks/issues/10327 to investigate.
